src/components/Button/Button.tsx

```diff
@@ -7,13 +7,22 @@ const Button = ({
   label,
   onClick,
   color = 'primary',
+  borderColor,
+  labelColor,
+  fontWeight,
   ...rest
 }: ButtonProps) => {
+  const inlineStyle: React.CSSProperties = {
```

Comment:
support css classname style attribute from parent component instead for the properties which are being passed directly to the component.  e.g. borderColor, fontWeight etc. 
Parent component can use the css classname props of the button and the styles are passed to the ButtonComp. 

e.g. 
Usages

<Button classname="text-sm, border-red" /> 

and in Button Component

<ButtonComp 
........
className = {...........   , props.className} >
.........
>